build/linux/dist/install.sh

```diff
@@ -205,6 +205,14 @@ simple_uninstall_f() {
 # Update desktop file and mime databases (if possible)
 updatedbs_f() {
 
+  if [ if "${HOME}/root/.config/mimeapps.list"]
```

Comment:
You have multiple syntax errors here. This line causes the error:
```
./install.sh: 208: [: missing ]
```
That is caused by the missing space before the closing bracket. Once that error is fixed, we get a new error:
```
./install.sh: 208: [: if: unexpected operator
```
Did you maybe mean to write [`-f`](http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html) instead of `if`?

Reply:
yes, sorry about that, I will change it now.